spec/factories/fitbit_bodies.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :fitbit_body do
+    weight '100'
+    bmi    '200'
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.